survey/README.md

```diff
@@ -0,0 +1,35 @@
+# The Package Manager Security Landscape Survey
+
+In 2022, the OpenSSF Securing Software Repositories Working Group embarked on
+an effort to survey the current security landscape of package manager
+ecosystems.
+
+This effort provides a survey/landscape of different security mechanisms and
+features that are implemented across the different ecosystems as it pertains to
```

Comment:
Nit: "as it pertains" -> "as they pertain"; the key word being "features", which is plural.